scripts/manual_assign.py

```diff
@@ -0,0 +1,55 @@
+import csv
+import re
+import requests
+import json
+
+
+def table_to_number(table):
+    letter = table[0].upper()
+    num = table[1:]
+    return (ord(letter) - 65) * 15 + int(num)
+
+
+# numeric value to table number: 210 -> N15, 1 -> A1
```

Comment:
Let's genericize this to also take in x tables (e.g. if layout adds another row of tables, it'll change from 15 to 16)